deepmd/pt/utils/env.py

```diff
@@ -5,14 +5,19 @@
 import torch
 
 from deepmd.env import (
+    GLOBAL_ENER_FLOAT_PRECISION,
+    GLOBAL_NP_FLOAT_PRECISION,
     get_default_nthreads,
     set_default_nthreads,
 )
 
-PRECISION = os.environ.get("PRECISION", "float64")
-GLOBAL_NP_FLOAT_PRECISION = getattr(np, PRECISION)
-GLOBAL_PT_FLOAT_PRECISION = getattr(torch, PRECISION)
-GLOBAL_ENER_FLOAT_PRECISION = getattr(np, PRECISION)
+numpy_to_torch_dtype_dict = {
```

Comment:
we may implement it by reversing dicts from str->np prec in dpmodel and str->torch prec.

Reply:
np->str can be simply written as `np.dtype(GLOBAL_NP_FLOAT_PRECISION).name`. I'll reuse `str->torch`.